extensions/volcano/tests/src/test/java/io/fabric8/volcano/test/V1Beta1VolcanoTest.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.volcano.test;
+import io.fabric8.volcano.client.VolcanoClient;
+import io.fabric8.volcano.scheduling.v1beta1.*;
+import io.fabric8.volcano.scheduling.v1beta1.PodGroupBuilder;
+import io.fabric8.volcano.scheduling.v1beta1.QueueBuilder;
+import io.fabric8.volcano.server.mock.EnableVolcanoMockClient;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@EnableVolcanoMockClient(crud = true)
+class V1Beta1VolcanoTest {
```

Comment:
CRUD mode tests are good but I would appreciate if you could add some expectations mode tests as well. They help in verifying that correct URLs are being hit and correct plural is being calculated by client.

Reply:
will add some https mode test